docs/source/serving/runai_model_streamer.rst

```diff
@@ -0,0 +1,50 @@
+.. _runai_model_streamer:
+
+Loading Models with Run:ai Model Streamer
+=========================================
+Run:ai Model Streamer is a library to read tensors in concurrency, while streaming it to GPU memory.
+Further reading can be found in `Run:ai Model Streamer Documentation <https://github.com/run-ai/runai-model-streamer/blob/master/docs/README.md>`_.
+
+vLLM supports loading weights in Safetensors format using the Run:ai Model Streamer.
+You first need to install vLLM RunAI optional dependency:
+
+.. code-block:: console
+
+    $ pip3 install vllm[runai]
+
+To run it as an OpenAI-compatible server, add the `--load-format runai_streamer` flag:
+
+.. code-block:: console
+
+    $ vllm serve /home/meta-llama/Llama-3.2-3B-Instruct --load-format runai_streamer
+
+To run model from AWS S3 object store run:
+
+.. code-block:: console
+
+    $ vllm serve s3://core-llm/Llama-3-8b --load-format runai_streamer
+
+
+To run model from a S3 compatible object store run:
+
+.. code-block:: console
+
+    $ RUNAI_STREAMER_S3_USE_VIRTUAL_ADDRESSING=0 AWS_EC2_METADATA_DISABLED=true AWS_ENDPOINT_URL=https://storage.googleapis.com vllm serve s3://core-llm/Llama-3-8b --load-format runai_streamer
+
+Tunable parameters
+------------------
+
+You can control the level of concurrency by using the `concurrency` parameter in `--model-loader-extra-config`:
```

Comment:
Can you elaborate a bit more about how does this parameter behave and what's the recommended way to tune it?

Reply:
I pointed the readers to further reading a few lines below
(https://github.com/vllm-project/vllm/blob/2c2b9f2c96142f21b4f995d7cc9dad67b8445af8/docs/source/serving/runai_model_streamer.rst?plain=1#L50)

Do you think we need to duplicate the detailed explanation here?
If you think so I will do it, 
I just prefer let the user read it first hand from the package description.

Reply:
That's a bit indirect. You could either summarize here or explicitly say "see details about tuning parameters in <URL>"

Reply:
Done! :)